Gruntfile.js

```diff
@@ -1,35 +0,0 @@
-var cheerio = require('cheerio')
-
-module.exports = function (grunt) {
-  grunt.loadNpmTasks('grunt-link-checker');
-
-  grunt.initConfig({
-    pkg: grunt.file.readJSON('package.json'),
-
-    linkChecker: {
```

Comment:
I'd like to keep this file (and its `package.json` dev dependencies), I still use this to check that there are no broken links when running locally and tweaking how links are parsed.